mlrun/projects/pipelines.py

```diff
@@ -932,6 +937,13 @@ def get_run_status(
             )
             run._exc = pipeline_runner_run.status.error
 
+            errors_counter = 0
+            if run._state == mlrun_pipelines.common.models.RunStatuses.succeeded:
+                errors_counter = 1
```

Comment:
1. The if should be `!=`
2. What about the other pipeline steps. This run is just the remote runner. You need to get all of the steps and sum the failed ones.